src/main/java/mate/academy/spring/service/Email.java

```diff
@@ -0,0 +1,22 @@
+package mate.academy.spring.service;

```

Comment:
It is better to create separate package for validation services.
```suggestion
package mate.academy.spring.validator;

```

Reply:
done